### DISCUSSION TITLE
Does webpack 5 support Node version 22 ?

### DISCUSSION THREAD

**Discussion by jashan777** [webpack/webpack#18406] [Q&A]
I was planning to upgrade my project to Node.js v22. Are there any potential issues or considerations I should be aware of regarding the compatibility of Webpack 5 with Node.js v22

**Comment by alexander-akait** [comment]
Yes, webpack support Node.js v22, we will add CI test today

**Reply by alexander-akait** [reply]
https://github.com/webpack/webpack/pull/18421
